### PR TITLE
feat(application-shell): fetch and expose user's created date for personalized guidance in mc

### PR DESCRIPTION
**Pull request by stephsprinkle**
#### Summary

The First Contact team is integrating Intercom into several applications within Merchant Center. In order to provide personalized guidance (pop ups, messaging, product tours) to new users, we need to provide Intercom with the user's created date. 

Screenshot 2023-01-06 at 9 43 29 AM

#### Description

- Fetching user's `createdAt` date
- Exposing user's `createdAt` date in application context

### CODE REVIEW

File: packages/permissions/src/types/generated/mc.ts [hunk 1]
```diff
@@ -312,6 +312,7 @@ export enum TPermissionScope {
   GetPermissionForAnyProject = 'get_permission_for_any_project',
   IntrospectOauthTokens = 'introspect_oauth_tokens',
   ManageApiClients = 'manage_api_clients',
+  ManageAssociateRoles = 'manage_associate_roles',
```
Comment (jaikamat) [inline packages/permissions/src/types/generated/mc.ts:315]: Were these included when the type generation script ran or do they relate to the `createdAt` work? Just curious

**Review by jaikamat — APPROVED** [review]
Looks straightforward!

File: packages/permissions/src/types/generated/mc.ts (hunk 1 above)
Comment (stephsprinkle) [reply packages/permissions/src/types/generated/mc.ts:315]: Apparently so!

**Review by CarlosCortizasCT — APPROVED** [review]
LGTM 👍

File: .changeset/slow-suns-greet.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+'@commercetools-frontend/application-shell': minor
+'@commercetools-frontend/application-shell-connectors': minor
+'@commercetools-frontend/permissions': minor
```
Comment (emmenko) [inline .changeset/slow-suns-greet.md:4]: Nit: I think `patch` is enough in this case.

```suggestion
'@commercetools-frontend/application-shell': patch
'@commercetools-frontend/application-shell-connectors': patch
'@commercetools-frontend/permissions': patch
```

**Review by emmenko — APPROVED** [review]
Thanks!

**Comment by stephsprinkle** [comment]
@emmenko @CarlosCortizasCT Not sure what to do about the failing checks, can either of you point me in the right direction?

**Comment by CarlosCortizasCT** [comment]
> @emmenko @CarlosCortizasCT Not sure what to do about the failing checks, can either of you point me in the right direction?

Hi @stephsprinkle ✋ 

This is not a problem related with this PR but with some repository configuration we need to update.
We're already looking into it 🧐 

Will let you know once we are finished 👍

**Comment by CarlosCortizasCT** [comment]
> > @emmenko @CarlosCortizasCT Not sure what to do about the failing checks, can either of you point me in the right direction?
> 
> Hi @stephsprinkle ✋
> 
> This is not a problem related with this PR but with some repository configuration we need to update. We're already looking into it 🧐
> 
> Will let you know once we are finished 👍

@stephsprinkle, issues with the PR checks have been addressed so this is good to merge.

Sorry for the inconveniences.

**Comment by stephsprinkle** [comment]
> > > @emmenko @CarlosCortizasCT Not sure what to do about the failing checks, can either of you point me in the right direction?
> > 
> > 
> > Hi @stephsprinkle ✋
> > This is not a problem related with this PR but with some repository configuration we need to update. We're already looking into it 🧐
> > Will let you know once we are finished 👍
> 
> @stephsprinkle, issues with the PR checks have been addressed so this is good to merge.
> 
> Sorry for the inconveniences.

Thank you! Not an inconvenience at all, I appreciate the review and assistance 😊

**Pull request merged**